theblues/plans.py

```diff
@@ -45,7 +45,7 @@ def get_plans(self, reference):
                 price=plan.get('price')), json))
         except (KeyError, TypeError, ValueError) as err:
```

Comment:
I think we should have another bare except clause to return a generic ServerError if some unexpected exception is raised.  Here and below.